### PR TITLE
add scim docs for okta [FF-2689]

### PR DESCRIPTION
**Pull request by leoromanovsky**
Documentation for configuration Okta with SCIM: 

https://deploy-preview-425--eppo-data-docs.netlify.app/administration/scim/

### CODE REVIEW

File: docs/administration/scim.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+sidebar_position: 4
+---
+
+# Provision Eppo users with SCIM
+
+:::note
+Provisioning users through SCIM is currently in closed Beta.
+:::
+
+## Configuring Directory Sync (SCIM)
```
Comment (emetelka) [inline docs/administration/scim.md:11]:
```suggestion
```

File: docs/administration/scim.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+Provisioning users through SCIM (the System for Cross-domain Identity Management) provides a secure and automated way to create and manage Eppo users through your preferred IdP (Okta, Microsoft Entra and others). We partner with [WorkOS](https://workos.com/docs/integrations/scim) to provide a secure SCIM connection using the 2.0 version of the SCIM protocol. Follow the steps from the WorkOS setup guide that you receive from your Eppo.
```
Comment (emetelka) [inline docs/administration/scim.md:13]:
```suggestion
Provisioning users through SCIM (the System for Cross-domain Identity Management) provides a secure and automated way to create and manage Eppo users through your preferred IdP (Okta, Microsoft Entra and others). We partner with [WorkOS](https://workos.com/docs/integrations/scim) to provide a secure SCIM connection using the 2.0 version of the SCIM protocol.
```

File: docs/administration/scim.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+
+Eppo offers support for:
+
+* Provisioning new users
+* Updating user profiles: name & role.
+* De-provisioning users
+
+### Eppo Application
```
Comment (emetelka) [inline docs/administration/scim.md:21]:
```suggestion
```

File: docs/administration/scim.md [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+
+Users created with SCIM can only be updated through your IdP.
```
Comment (emetelka) [inline docs/administration/scim.md:23]:
```suggestion
:::note
Users created with SCIM can only be updated through your IdP.
:::
```

File: docs/administration/scim.md [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+Fill out the fields as shown in the screenshot above: Paste the base URL from the WorkOS setup guide into the SCIM connector base URL field. 
+
+* For Unique identifier field for users, set to "email". 
+* Check the Push New Users and Push Profile Updates. 
+* For Authentication Mode, select HTTP Header and paste the Bearer Token from the WorkOS set up to the field. 
+* Click Save.  
+
+### Attributes and Roles on Okta
+
+Set up Attribute Mapping, you will see that we support a custom attribute called `eppoMemberRole`. This optional custom attribute allows an IT admin to set the user's Eppo role from within the IdP.
```
Comment (emetelka) [inline docs/administration/scim.md:51]:
```suggestion
To setup Attribute Mapping, you will see that we support a custom attribute called `eppoMemberRole`. This optional custom attribute allows an IT admin to set the user's Eppo role from within the IdP.
```

**Review by emetelka — COMMENTED** [review]
Made small edits. The okta section is much more featured than the Entra section. Is that intentional - is Entra that much easier?

**Comment by btthomas** [comment]
> Made small edits. The okta section is much more featured than the Entra section. Is that intentional - is Entra that much easier?

Aren't we missing a step for Entra to copy+paste the endpoint, bearer token, and to set `email` as the user identifier?

**Review by greghuels — APPROVED** [review]
This looks good. My only feedback is that you could dive into how you'd sync existing users once Okta/Entra is integrated with WorkOS.

**Comment by leoromanovsky** [comment]
> Made small edits. The okta section is much more featured than the Entra section. Is that intentional - is Entra that much easier?



> This looks good. My only feedback is that you could dive into how you'd sync existing users once Okta/Entra is integrated with WorkOS.



> > Made small edits. The okta section is much more featured than the Entra section. Is that intentional - is Entra that much easier?
> 
> Aren't we missing a step for Entra to copy+paste the endpoint, bearer token, and to set `email` as the user identifier?

Sure I can be verbose - what's going to happen is IT admins will receive an email from WorkOS and it will have a wizard tailor to entra/okta/etc. Theirs is probably better than anything we can write.

However we do "own it" and can't rely on it so I can bump up the detail level to this guide.

**Pull request merged**